### PR TITLE
Whitelabel updates based on feedback and other edits

### PR DESCRIPTION
**Pull request by ksigler7**

### CODE REVIEW

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.md [hunk 1]
```diff
@@ -74,23 +76,31 @@ Setting up domain authentication
...
 1. Fill in the domain that you want to send from and add advanced settings as needed. Make sure that you only enter the name of your root domain. Do not include `www` or `http://www` in this field! Your domain needs to match the domain of your FROM address on the emails you are sending out. For example, if I am sending an email from `example@sendgrid.com`, I would set my domain authentication domain to be `sendgrid.com`. Click **Next**. For more information about advanced settings, see [Advanced settings](#-Advanced-settings).
 1. Next, you need to add all of the CNAME records on this screen to your DNS host. This process varies depending on your DNS host. For videos on how to add your CNAME to some popular DNS service providers, check out these [videos](https://sendgrid.com/docs/User_Guide/Settings/Whitelabel/providers.html). If you don't have access to modify your companies DNS records, you can also email a request to a co-worker. This email includes a direct link to the CNAME records that the recipient of the email can access for 48 hours [DOUBLE CHECK]. The recipient doesn't need login access to your SendGrid account.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.md]: remove "double check" note at the end of this section.

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.md [hunk 2]
```diff
@@ -30,9 +32,9 @@ navigation:
...
+Even though this is a small change from your recipients perspective, this change has a huge positive impact on your reputation as a sender and your email deliverability. Email service providers distrust messages that don't have domain authentication set up because they can not be sure that the message comes from you. Explicitly stating that it comes from you increases your reputation with email service providers which makes it much less likely that they will filter your mail and not allow it get to your recipient's inbox, which increases your deliverability. You are also explicitly showing your recipients that this email comes from you, so they are less likely to mark your mail as spam.
 
 {% anchor h2 %}
 Key terminology
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.md]: It won't let me comment right on the entry, but you use the abbreviation "ISP" in the DKIM entry. Is that what we told Academy people to use? I can't remember and I want to keep it consistent.

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md [hunk 3]
```diff
@@ -80,7 +68,7 @@ Note that you can also set up link branding when you set up your domain authenti
...
+1. Enter the domain that you want to brand the links and images with and add advanced settings. Make sure that you only enter the name of your root domain. Do not include `www` or `http://www` in this field! Your domain needs to match the domain of your from address on the emails you are sending out. For example, if I am branding with the domain `example.sendgrid.com`, I would set my link branding domain to be `sendgrid.com`. Click **Next**. For more information about advanced settings, see [Advanced settings](#-Advanced-settings).
 1. Next, you need to add all of the CNAME records on this screen to your DNS host. This process varies depending on your DNS host. For videos on how to add your CNAME to some popular DNS service providers, check out these [videos](https://sendgrid.com/docs/User_Guide/Settings/Whitelabel/providers.html). If you don't have access to modify your companies DNS records, you can also email a request to a co-worker. This email includes a direct link to the CNAME records that the recipient of the email can access for 48 hours [DOUBLE CHECK]. The recipient doesn't need login access to your SendGrid account.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md]: Delete "double check" note at the end of this step.

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md [hunk 4]
```diff
@@ -91,25 +79,31 @@ It can take up to 48 hours for the records to verify after you upload them into
 Verifying your DNS
 {% endanchor %}
 
-Once you add the CNAME records to your DNS host, return to the [Sender authentication page](https://app.sendgrid.com/settings/whitelabel) and click **Verify**. If you click verify, and only half of your CNAME records verify, this usually means that you need to wait a bit longer.
+Once you add the CNAME records to your DNS host, return to the [Sender authentication page](https://app.sendgrid.com/settings/whitelabel) and click **Verify**.
 
-Any time that you send an email with a FROM address whose domain matches the domain set in the domain authentication, SendGrid applies that domain to your email. You only need to update your domain authentication if you want to update the domain you are emailing from.
+{% info %}
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md]: stray info tag?

File: source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md [hunk 5]
```diff
@@ -91,25 +79,31 @@ It can take up to 48 hours for the records to verify after you upload them into
...
+*To customize your subdomain:*
+
+When you are in the process of branding a link, and on the screen where you input domain settings, open the advanced settings, select **Custom subdomain for links** and input letters or numbers to build a custom domain. If you don't select these, SendGrid automatically selects them for you. Make sure the characters you select are different from what SendGrid assigned you initially.
 
 {% anchor h3 %}
 Assigning a subuser
 {% endanchor %}
 
-By assigning a domain whitelabel to one of your subusers, you can give them the benefit of improved authentication and security, but also separate from the sending reputation of your parent account. If you authenticate a subusers domain, they can't edit or delete it.
+By assigning a domain whitelabel to one of your subusers, you can give them the benefit of improved authentication and security, but also separate from the sending reputation of your parent account. If you assign a subusers domain, they can't edit or delete it.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.md]: I see the word 'whitelabel' in the first sentence. 🙃

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**